circuits/cpp/barretenberg/ts/package.json

```
@@ -21,18 +21,19 @@
   ],
   "scripts": {
     "clean": "rm -rf ./dest .tsbuildinfo.browser .tsbuildinfo.node",
-    "build": "yarn clean && yarn build:wasm && yarn build:ts:browser && yarn build:ts:node",
+    "build": "yarn clean && yarn build:wasm && yarn copy_wasm_dest && yarn build:ts:browser && yarn build:ts:node",
     "build:dev": "tsc -b --watch",
     "build:wasm": "cd ../cpp && cmake --preset wasm-threads && cmake --build --preset wasm-threads && cmake --preset wasm && cmake --build --preset wasm",
+    "copy_wasm_dest": "cp ../cpp/build-wasm/bin/barretenberg.wasm ./dest/ && cp ../cpp/build-wasm-threads/bin/barretenberg.wasm ./dest/barretenberg-threads.wasm",
     "build:ts:browser": "tsc -b tsconfig.browser.json && BUILD_TARGET=browser webpack && chmod +x ./dest/browser/main.js && BUILD_TARGET=browser node replace_imports.cjs",
     "build:ts:node": "tsc -b tsconfig.node.json && chmod +x ./dest/node/main.js && BUILD_TARGET=node node replace_imports.cjs",
     "build:bindings": "cd .. && ./scripts/bindgen.sh",
     "serve": "webpack serve",
     "formatting": "prettier --check ./src && eslint --max-warnings 0 ./src",
     "formatting:fix": "prettier -w ./src",
-    "test": "yarn build:ts:browser && yarn build:ts:node && yarn test:jest && yarn test:bin",
+    "test": "yarn build:ts:browser && yarn build:ts:node && yarn test:jest && yarn test-bin",
     "test:jest": "NODE_NO_WARNINGS=1 node --experimental-vm-modules $(yarn bin jest) --no-cache --passWithNoTests",
-    "test:bin": "cd ./bin-test && ./bin-test.sh",
+    "test-bin": "bin-test/test.js",
```

```suggestion
    "test:bin": "bin-test/test.js",
```
We should be consistent with other commands.